examples/text_classification/model.py

```diff
@@ -15,4 +25,10 @@ def init_weights(self):
         self.fc.bias.data.zero_()
 
     def forward(self, text, offsets):
+        r'''
+        Arguments:
+            text: a bag of text tensors
+            offsets: a list of offsets
```

Comment:
I'd mention that these are offsets that delimit the individual sequences within the "bag"